File: c/1060.c
#include <stdio.h>

int main(){
    double e, cont=0;

    for (int i=0;i<6;i++){
        scanf("%lf", &e);
        if (e>0){
            cont++;
        }
    }
    printf("%.0lf valores positivos\n", cont);
}

File: js/PROBLEMS/1060.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8')
var lines = input.split('\n')

lines = lines.map(function (item){
    return parseFloat(item)
  })

cont = 0;
for (i=0;i<lines.length;i++){

    if (lines[i] > 0){
        cont++;
    }
}
console.log(""+ cont +" valores positivos")